crates/bevy_pbr/src/render/mesh_vertex_output.wgsl

```diff
@@ -0,0 +1,11 @@
+#define_import_path bevy_pbr::mesh_vertex_output
```

Comment:
I like this and had done something similar on a branch locally at some point. Good choice! :)